From d783c2a8869b6289381ec860c9dfe58bbc9d6eac Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Tue, 12 Jan 2016 15:12:03 +0800
Subject: [PATCH 0831/5242] MLK-12211-1: ARM: dts: imx6/imx7: enable
 snvs-poweroff

commit  d94b657607f22db390ed3bf380f9c7d447f4fb38 from
https://source.codeaurora.org/external/imx/linux-imx.git

enable snvs-poweroff driver on imx6sx(except sabreauto board),imx6ul and
imx7d all boards.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-14x14-arm2.dts |    4 ++++
 arch/arm/boot/dts/imx6sx-19x19-arm2.dts |    4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-14x14-arm2.dts b/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
index 8af90f1..cf9809f 100644
--- a/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
@@ -464,6 +464,10 @@
 	status = "okay";
 };
 
+&snvs_poweroff {
+	status = "okay";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1_1>;
diff --git a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
index 64ac72d..4460742 100644
--- a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
@@ -433,6 +433,10 @@
 	status = "disabled";
 };
 
+&snvs_poweroff {
+	status = "okay";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1_1>;
-- 
1.7.9.5

